packages/framework/esm-api/src/types/fhir-resource.ts

```diff
@@ -22,6 +22,7 @@ export interface FHIRResource {
     };
     valueString: string;
     valueCodeableConcept: { coding: Array<FHIRCode> };
+    valueDateTime: string;
```

Comment:
This change will be made through the whole app, might raise issues at various places.

```suggestion
    valueDateTime?: string;
```

Reply:
@vasharma05 Done!
Thanks